frontend/src/app/patients/personSchema.ts

```diff
@@ -158,6 +159,37 @@ export function isValidBirthdate18n(t: TFunction) {
   };
 }
 
+function isValidEmail18n(t: TFunction) {
+  return function isValidEmail(this: TestContext, email: string | undefined) {
```

Comment:
return anonymous arrow function

Reply:
Is this a preference thing or is there a footgun I don't know? Anonymous functions are great for a lot of things, but I think names are perfectly sensible for returning functions that may be called directly!

```
function named() {
  return function namedFn() {
    throw new Error('I have a name');
  }
}

function anon() {
  return () => {
    throw new Error('I have no name');
  }
}

const n = named();
const a = anon();

try {
  n();
} catch(e) {
  console.log(e);
}

try {
  a();
} catch(e) {
  console.log(e);
}
```

```
➜  prime-simplereport git:(nac/2848-bugix-device-update) ✗ node things.js
Error: I have a name
    at namedFn (/Users/nathancarter/repos/prime-simplereport/things.js:3:11)
    at Object.<anonymous> (/Users/nathancarter/repos/prime-simplereport/things.js:17:3)
Error: I have no name
    at /Users/nathancarter/repos/prime-simplereport/things.js:9:11
    at Object.<anonymous> (/Users/nathancarter/repos/prime-simplereport/things.js:23:3)
```

Reply:
yeah, they're basically equivalent, but with returning a named function, the name is never used, thus it serves no purpose, hence why anonymous functions are a thing.

Reply:
I am not sure what you mean by `"I think names are perfectly sensible for returning functions that may be called directly!"`,  `isValidEmail` is not accessible directly. you have to use `isValidEmail18n`